src/components/QuestDetails/components/StreakProgress/index.tsx

```diff
@@ -152,14 +144,10 @@ export default function StreakProgress({
             {` / ${requiredStreakInDays} ${i18n.days}`}
           </div>
         </div>
-        <div className={styles.progressIconsContainer}>
-          {lightningBoltIcons.map((filled, index) => (
-            <LightningBolt
-              className={classNames(styles.streakIcon, getFilledClass(filled))}
-              key={`lightningBolt:${index}`}
-            />
-          ))}
-          <LightningBolt />
+        <div className={styles.syncContainer}>
+          <Button type="secondaryGradient" onClick={onSync}>
```

Comment:
the size here is small in the design. 
![image](https://github.com/user-attachments/assets/0656c6ef-eb6c-416e-aecc-805c7939078f)